### PR TITLE
add noble testnet

### PR DESCRIPTION
**Pull request by boojamya**
This PR...

- Adds the `grand-1` Noble testnet.
- Adds`packet-forward-middleware` as a feature to both noble `mainnet` and `testnet`.
- Add some `coinImageUrl`'s

### CODE REVIEW

**Comment by HeesungB** [comment]
Hello @boojamya !
There are a few things to change.
 - Noble testnet's chainIdentifer is `grand`, so Could you change file name from `noble-testnet.json` to `grand.json`?
 - The token image should be named `coinMinimalDenom`. Could you change file name?
 - Image size should be 256x256px size. Could you resize image?
 - There is no feature named ` "packet-forward-middleware"`, Could you remove this feature?

File: cosmos/noble.json [hunk 1]
```diff
@@ -52,6 +56,7 @@
       "coinDenom": "ATOM",
       "coinMinimalDenom": "ibc/EF48E6B1A1A19F47ECAEA62F5670C37C0580E86A9E88498B7E393EB6F49F33C0",
       "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cosmoshub/uatom.png",
```
Comment (HeesungB) [inline cosmos/noble.json:59]: @boojamya IBC tokens do not require you to add an image link.
Could you remove this field?

**Comment by HeesungB** [comment]
@boojamya And Could you rename file name from `grand-testnet.json` to `grand.json`?
User can see chain name on Keplr Extension.

File: cosmos/grand.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  "chainId": "grand-1",
+  "chainName": "Noble",
+  "rpc": "https://rpc.testnet.noble.strange.love",
+  "rest": "https://api.testnet.noble.strange.love/",
+  "nodeProvider": {
+    "name": "Strangelove",
+    "email": "support@strange.love",
+    "website": "https://strange.love/"
+  },
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/noble/chain.png",
```
Comment (HeesungB) [inline cosmos/grand.json:11]: @boojamya 
The chain image file should go in the folder of that chainIdentifier!
So, Could you make a directory `images/grand` and add `chain.png` image file.
after that Could you change `chainSymbolImageUrl` field?

Comment (boojamya) [reply cosmos/grand.json]: Done! 
Just pointing out the obvious here, but grand-1 is the testnet version of noble-1 so they are nearly identical. 

lmk if there is anything else lol.

**Comment by HeesungB** [comment]
@boojamya 
Could you add token image too?, about `currencies` `feeCurrencies` field In `grand.json`.
`coinImageUrl` will be `grand/uusdc.png`, not `noble/uusdc.png`

And you can test on localhost use below command
```
 yarn validate cosmos/grand.json
 ```

**Comment by HeesungB** [comment]
@boojamya Thank you for contribution!

**Pull request merged**